tests/skip.rs

```diff
@@ -92,3 +92,34 @@ fn skip_enum() {
         }
     );
 }
+
+#[test]
+fn skip_help_doc_comments() {
+    #[derive(StructOpt, Debug, PartialEq)]
+    #[structopt(name = "a")]
+    pub struct Opt {
+        #[structopt(skip, help = "internal_stuff")]
+        a: u32,
+
+        #[structopt(skip, long_help = "internal_stuff\ndo not touch")]
+        b: u32,
+
+        /// Not meant to be used by clap.
+        /// I want a default here.
```

Comment:
Did you check the test fail without the fix? Because I think an empty line is needed at the second line to generate a long_help

Reply:
Yep, forgot to change